src/test/java/bio/terra/common/BaseTest.java

```diff
@@ -100,6 +102,13 @@ public void assertStorageObjectData(StorageObjectData data) {
     assertEquals("value2", data.getMetadata().get("key2"));
   }
 
+  public void assertGCSEvent(GCSEvent data) {
+    // Check String deserialization
+    assertEquals("terra-kernel-k8s-testrunner-results", data.getBucket());
+    // Check OffsetDateTime deserialization
+    assertEquals("2021-07-07T22:57:14.257Z", data.getTimeCreated().toInstant());
+  }
+
   public void verifyMockTGZArchiveEntry(String filename, long bytes) {
     System.out.println("Verifying " + filename + " filesize.");
     if (filename.contains("RENDERED")) {
```

Comment:
`verifyMockTGZArchiveEntry` needs an `else { fail(… }` clause. If some test calls this method with malformed data - where the filename doesn't contain any of the magic strings - this method will PASS, because it'll never run any assertions.

Reply:
Added `fail` method to detect malformed data.